static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -29,6 +33,17 @@ function ReplayMetaData({replayRecord}: Props) {
     },
   };
 
+  const errorCountByProject = useMemo(
```

Comment:
we need `useMemo` when there's only expected to be like single-digits number in the aray?
the header can't really scale more to render more than that.

Reply:
i don't quite understand; but if we're saying that `useMemo` here is an over optimization, yes it is lol